### ISSUE TITLE
Great Wall of China scenario has unusually high initial cash, leading to unusually high company value upon completion

### ISSUE THREAD

**Issue by SpartanFrederic104** [OpenRCT2/OpenRCT2#17984]
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.1-250-ga5a6d3a (a5a6d3a on develop) provided by GitHub

### Describe the issue

![Great Wall of China 2022-09-07 08-45-47](https://user-images.githubusercontent.com/61885939/188924565-31523602-5a26-427f-b021-dbf6ffd5276d.png)
The Asia - Great Wall of China Tourism Enhancement scenario from the Wacky Worlds expansion pack has its initial cash set to **$5,010,000**, which is extremely high and way beyond the maximum possible initial cash of $1,000,000 and is also very likely an error. Because of this, completing that scenario will always result in an unusually high company value above $5,000,000, which shouldn't otherwise be possible in a normal scenario outside of cheats, playing for a very long period of time and earning millions in both cash and park value, or by spamming a lot of ultra-extreme intensity rides to increase the park value significantly.

### Steps to reproduce

1. Start up Great Wall of China scenario
2. Enable money in it
3. Complete the scenario

**Comment by 733737** [comment]
is this really a bug??

**Comment by LordMarcel** [comment]
This seems like a conscious choice by Chris Sawyer, similar to the multi-million dollar loans you start with in some real park scenarios.

**Comment by Gymnasiast** [comment]
Not really, as this scenario has money disabled. I think the assumption is that a "no money" scenario will always have £0 cash.

Should be easy to fix by first checking if money is enabled. If not, assume cash is 0 and only count the park value.
